### PR TITLE
feat(my-pages): Mileage charts for vehicles

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

* Add charts to vehicle mileage screen and the bulk vehicle mileage screen

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
  - Introduced a new feature flag for vehicle bulk mileage subdata page.
  - Added new messages for vehicle-related functionalities, enhancing localization.
  - New `VehicleBulkMileageSubData` component for displaying mileage registration history.
  - New `SimpleBarChart` component for visualizing data.
  - Enhanced `EmptyTable` component with a customizable background property.
  - Added new chart-related components and interfaces for improved data visualization.

- **Bug Fixes**
  - Updated existing messages for improved clarity and accuracy.

- **Style**
  - Added new styles for chart components to enhance visual presentation.

- **Documentation**
  - Expanded export statements to include new components and messages.

### CODE REVIEW

**Review by magnearun — APPROVED** [review]
core files lgtm :)

File: libs/portals/my-pages/assets/src/screens/VehicleBulkMileage/VehicleBulkMileageTable.tsx [hunk 1]
```diff
@@ -32,13 +65,46 @@ const VehicleBulkMileageTable = ({ vehicles, loading }: Props) => {
...
             />
-            <T.Body>{rows}</T.Body>
+            <T.Body>
+              {rows}
+              {rows.length > 0 && (
+                <T.Row>
+                  <td>
+                    <Box marginTop={2}>
+                      <Text variant="medium" fontWeight="semiBold">
+                        Samtals:
```
Comment (thordurhhh) [inline libs/portals/my-pages/assets/src/screens/VehicleBulkMileage/VehicleBulkMileageTable.tsx:81]: Translation?

File: libs/portals/my-pages/assets/src/screens/VehicleBulkMileage/VehicleBulkMileage.tsx [hunk 2]
```diff
@@ -143,62 +148,68 @@ const VehicleBulkMileage = () => {
...
+          {displayFilters && (
+            <Box marginBottom={2}>
+              <Filter
+                labelClear={formatMessage(m.clearFilter)}
+                labelClearAll={formatMessage(m.clearAllFilters)}
+                labelOpen={formatMessage(m.openFilter)}
+                labelClose={formatMessage(m.closeFilter)}
+                variant="popover"
+                onFilterClear={() => {
+                  console.log('clear')
```
Comment (thordurhhh) [inline libs/portals/my-pages/assets/src/screens/VehicleBulkMileage/VehicleBulkMileage.tsx:181]: console log

**Review by thordurhhh — CHANGES_REQUESTED** [review]
minor adjustments

**Review by thordurhhh — APPROVED** [review]
Nice!

**Pull request merged**